packages/now-node/src/index.ts

```diff
@@ -104,12 +104,7 @@ async function downloadInstallAndBundle({
   } else {
     const installTime = Date.now();
     console.log('Installing dependencies...');
-    await runNpmInstall(
-      entrypointFsDirname,
-      ['--prefer-offline'],
-      spawnOpts,
-      meta
-    );
+    await runNpmInstall(entrypointFsDirname, [], spawnOpts, meta);
```

Comment:
This is missing the `config.installCommand`

Reply:
`installCommand` is for frontend frameworks, not serverless functions.